manage/heroes/new.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+require '../layout/header.php';
+
+use Model\hero;
+use Service\Container;
+use Model\AbstractShip; // get the teams
+
+$teams = AbstractShip::getTeams();
```

Comment:
Get the teams from Hero. See note in Hero.php

Reply:
Getting teams from the Hero class with an alias of ModelHero